lib/chef/resource/sysctl.rb

```diff
@@ -45,6 +45,10 @@ class Sysctl < Chef::Resource
         coerce: proc { |v| coerce_value(v) },
         required: true
 
+      property :comment, [Array, String],
+        description: "Comments, placed above the resource setting in the generated file. For multi-line comments, use an array of strings, one per line.",
```

Comment:
Descriptions for properties should ideally be full sentences. Please change the first sentence to include a verb. For example, "Comments _are_ placed above the resource setting in the generated file."? Feel free to rework/insert your own verb.

Reply:
FYI The reason why these descriptions are so important is because they are aggregated from this file and then displayed on our docs website.